.github/workflows/code-freeze.yml

```diff
@@ -0,0 +1,153 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Code Freeze Bot
+on: workflow_call
+
+env:
+  PMC_MEMBERS: '[ "gdams", "sxa", "johnoliver", "tellison", "jerboaa", "smlambert", "karianna", "llxia", "sanhong" ]'
```

Comment:
Can this be grabbed from Github team membership API?

Reply:
There is no GitHub team for PMC right now